README.md

```diff
@@ -78,3 +79,55 @@ applications are running the functional tests can be run with
 `make run`
 
 (or you can substitute `local` with the name of whatever `config/*.sh` environment file you're using.
+
+
+## Setting up your local environment with Docker Compose to run functional tests against
+
+Setting up your local environment and database to run the functional tests against can be a pain. Wouldn't it be nice if you could get all the apps up and running, backed by a database in the correct state with just one command? That's hopefully what we can do with Docker Compose. It uses the latest tagged Docker images of the apps as well as a DB dump of your choosing to achieve this.
+
+### Setup
+
+  - Have Docker installed on your machine - see [here](https://www.docker.com/docker-mac)
+  - Have Docker Compose installed on your machine. Docker for Mac comes with it so you probably won't need to do anything extra. You can check with `docker-compose -v`. If you don't have it, go [here](https://docs.docker.com/compose/install/)
+  - Have a `.envrc` file in the root of your credentials repo which sets your AWS profile to your sops profile. Probably something like `AWS_PROFILE=sops`. Check in `~/.aws/config` if you're not sure. This step isn't strictly necessary to get all the containers up and running, but functional tests will fail whenever trying to send an email if you don't do it.
+  - Have a database dump that you want to use. The most up to date you can get your hands on the better. This is the data you're going to be running tests against so it should have all frameworks up to date.
+  - Place your database dump into the `./sql` directory.
+  - Rename your dump to `database_dump`. This needs to be exact - no suffix!
```

Comment:
Is that because postgres will pick up any sql files in `/docker-entrypoint-initdb.d`? I think if you map sql folder to a different directory you could modify the script to run the file from there without requiring a name. It's also very likely that it would be called `exportdata.sql` since that's what our DB export files are names.